app/org/maproulette/models/dal/TaskReviewDAL.scala

```diff
@@ -420,7 +469,7 @@ class TaskReviewDAL @Inject()(override val db: Database,
     val query = user.isSuperUser match {
       case true =>
         s"""
-          SELECT tasks.${this.retrieveColumnsWithReview} FROM tasks
+          SELECT t.${this.retrieveColumnsWithReview} FROM tasks
```

Comment:
Since you're switching to `t` I think you need `FROM tasks t`

Reply:
Yeah that is a mistake, I switched to t and then realized that was not going to work, obviously I didn't revert every place it happens. Thanks for catching it.